src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricSerializer.cs

```diff
@@ -50,9 +74,22 @@ public void SerializeMetric(StringBuilder sb, Metric metric)
         {
             foreach (var metricData in metric.Data)
             {
-                WriteMetricKey(sb, metric);
-                WriteDimensions(sb, metricData.Labels);
-                WriteDimensions(sb, this._defaultDimensions);
+                var metricKey = CreateMetricKey(metric);
+                // skip lines with invalid metric keys.
+                if (string.IsNullOrEmpty(metricKey))
+                {
+                    _logger.LogWarning("metric key was empty after normalization, skipping metric (original name {})", metric.MetricName);
```

Comment:
```suggestion
                    _logger.LogWarning("metric key was empty after normalization, skipping metric (original name '{}')", metric.MetricName);
```
Makes it easier to spot if it was empty in first place or the like.

Reply:
I'd say it's also common to put in the message template the name of the argument. These then show up and it's easier to see what the argument is. For ex: `_logger.LogInformation(LoggingEvents.GetItem, "Getting item {Id}", id);` In this case I would add `(original name '{metricName}')`